README.md

```diff
@@ -511,6 +511,8 @@ These settings can be changed at any time.
 * `settings.ecs.allow-privileged-containers`: Whether launching privileged containers is allowed on the container instance.
   If this value is set to false, privileged containers are not permitted.
   Bottlerocket sets this value to false by default.
+* `settings.ecs.container-stop-timeout`: Time to wait for the task's container to stop on their own before they are forcefully stopped.
+Valid time units are `ns`, `µs`, `ms`, `s`, `m`, and `h`, e.g. `1h`, `1m1s`
```

Comment:
nit: if there are values that are too small for the `ecs-agent` to bother with, we could omit them:
```suggestion
Valid time units include `s`, `m`, and `h`, e.g. `1h`, `1m1s`
```